src/Sylius/Bundle/ApiBundle/CommandHandler/Catalog/AddProductReviewHandler.php

```diff
@@ -36,9 +37,13 @@ public function __construct(
 
     public function __invoke(AddProductReview $addProductReview): ReviewInterface
     {
-        /** @var ProductInterface $product */
+        /** @var null|ProductInterface $product */
```

Comment:
```suggestion
        /** @var ProductInterface|null $product */
```
Pretty sure it's noted somewhere is Symfony's coding style that when annotating types null should always be at the end